tool/llpcAutoLayout.cpp

```diff
@@ -461,10 +468,25 @@ void DoAutoLayoutDesc(
         default:
             {
                 SPIRVWord binding = SPIRVID_INVALID;
-                SPIRVWord descSet = SPIRVID_INVALID;
-                if (pVar->hasDecorate(DecorationBinding, 0, &binding) &&
-                    pVar->hasDecorate(DecorationDescriptorSet, 0, &descSet))
+                SPIRVWord descSet = 0;
+                if (pVar->hasDecorate(DecorationBinding, 0, &binding))
                 {
+                    // This copes with a variable having a binding but no set. I saw that happen in
```

Comment:
Change the comment:

// NOTE: Test "shaderdb/OpDecorationGroup_TestGroupAndGroupMember_lit.spvas" defines a variable having a binding but no set. We should handle this case.